### DISCUSSION TITLE
Create calculated field which can be changed

### DISCUSSION THREAD

**Discussion by JakovGlavac** [react-hook-form/react-hook-form#6221] [Q&A]
Hi. I'm trying to create form where one of two fields can be changed, and on change other will be calculated, so my question is how to create something like [this](https://www.omnicalculator.com/other/earthquake)?

**Comment by bluebill1049** [comment]
the question is too generic, can you be more specific?

**Reply by JakovGlavac** [reply]
Sorry for not answering in a while, but in short I would like to update other fields with calculated value, when one filed changes. Like in this video. 
https://user-images.githubusercontent.com/34971091/129891770-e8e0f86c-e589-4bd8-ad7d-da28b713498e.mp4

When I attempted to do it, I created infinite loop where one filed would update and than trigger other filed to update other and so fort.
